_data/navbars/maintenance.yml

```diff
@@ -11,6 +11,8 @@ section:
     path: /maintenance/openshift-upgrade
   - title: OpenStack
     path: /maintenance/openstack-upgrade
+- title: Migrate Calico data
```

Comment:
I don't know if this is the right phrasing of this feature.

Maybe "Migrate datastore from etcd to Kubernetes"? Or "Change which datastore is in use"?